rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form.json

```diff
@@ -793,6 +834,28 @@
           "err": "{{patient-diagnostics-form-original.step2.has_muscle_pains.v_required.err}}"
         }
       },
+      {
+        "key": "has_abdominal_pain",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "native_radio",
+        "label": "{{patient-diagnostics-form-original.step2.has_abdominal_pain.label}}",
+        "options": [
+          {
+            "key": "Yes",
```

Comment:
Could we make the keys lowercase for uniformity?

Reply:
fixed.